pkg/models/address_list.go

```diff
@@ -4,13 +4,77 @@
 
 package models
 
-import "time"
+import (
+	"time"
+
+	"openpitrix.io/notification/pkg/constants"
+	"openpitrix.io/notification/pkg/pb"
+	"openpitrix.io/notification/pkg/util/idutil"
+	"openpitrix.io/notification/pkg/util/pbutil"
+)
 
 type AddressList struct {
-	AddressListId string    `gorm:"column:address_list_id"`
-	Name          string    `gorm:"column:name"`
-	Extra         string    `gorm:"column:extra"`
-	Status        string    `gorm:"column:status"`
-	CreateTime    time.Time `gorm:"column:create_time"`
-	StatusTime    time.Time `gorm:"column:status_time"`
+	AddressListId   string    `gorm:"column:address_list_id"`
+	AddressListName string    `gorm:"column:address_list_name"`
+	Extra           string    `gorm:"column:extra"`
+	Status          string    `gorm:"column:status"`
+	CreateTime      time.Time `gorm:"column:create_time"`
+	StatusTime      time.Time `gorm:"column:status_time"`
+}
+
+const (
+	AddressListIdPrefix = "adl-"
+)
+
+//table name
+const (
+	TableAddressList = "address_list"
+)
+
+//field name
+//AddrLs is short for Address.
+const (
+	AddrLsColId      = "address_list_id"
```

Comment:
ColumnAddressListId

no need for short.  If the vals are column name, add Column as prefix.